tls/s2n_tls13_secrets.c

```diff
@@ -100,6 +100,28 @@ S2N_RESULT s2n_tls13_empty_transcripts_init()
     return S2N_RESULT_OK;
 }
 
+static S2N_RESULT s2n_set_transcript_hash(struct s2n_connection *conn)
```

Comment:
Shouldn't this be called something like "s2n_calculate_transcript_digest"? I liked the old name.